team-telnyx/telnyx-webrtc-android
# Feature/webrtc 2358 stats

[WebRTC-2358 - Call debug stats](https://telnyx.atlassian.net/browse/WEBRTC-2358)

---
## What was before
- no support for debug mode in SDK
- no stats debug data generated

### After changes
- support for debug mode in SDK
- when enabled on a call, all of the statics types are shipped from the SDK client to the voice debug service

## ✋ Manual testing
1. in demo app build.gradle set IS_STATS_DEBUG_MODE to 'true'
2. start or accept a call
3. end call
4. check peerId which is visible in app logs
5. grab generated debug file from portal.telnyx.com and push it to http://webrtc-debug.telnyx.com.us-west-1.telnyxstorage.com/index.html
